Project-Progress.md

```diff
@@ -0,0 +1,27 @@
+# Milestone 2
+
+# Project Realization - Overview
+
+Initially in Milestone 1, we planned on gathering the requirements that helps in building the appropriate backend and frontend environments. Based on the meeting with Omaha STEM Ecosystem as a team we decided to follow agile methodology. Each week we complete set of tickets on sprint basis.
+
+As a team of 4 members, we divided work and has been working on two different portions of the project.
+
+The major portion of the project (**Part 1**): Two team members worked on the ground work to build the environment for the frontend and backend based on the requirements we gathered from Omaha STEM Ecosystem.
+
+The second portion of the project (**Part 2**): Started to work on the functionality (Home page with Nav bar, Registering Users page and Login) parts of the project.
+
+# Progress:
+
+The team has almost finished the major chunk (Part 1) of the project. Over the earlier weeks, they worked on completing every part to build backend based angular framework and frontend is based on Django REST framework and docker is used to run and deploy our web application using docker container. The ultimate major reason of choosing these environments is to build a secure web application.
```

Comment:
The frontend and backend frameworks are backwards. The backend is Django and frontend is Angular.

Reply:
Ohh my bad! Sorry, its typo. Corrected it.